tasks/task02/Area_of_a_triangle.py

```diff
@@ -0,0 +1,42 @@
+# main menu
+import math
+print("Welcome to the triangle area calculation tool.")
+while True:
+    print("\n","Menu:","1. Calculate triangle area by base and height","2. Calculate triangle area by 2 sides and angle between them","3. Exit", sep = "\n")
+    print("Enter menu item number: ", end = "")
+    mitem = input()
+    if mitem == "1":
+        print("Enter base and height (space separated): ", end = "")
+        while True:
+            inp = input().split(" ")
+            try:
+                base, height = [float(x) for x in inp]
+                print("\nbase =", base, " height =", height) 
+                print("Area is: ", base * height / 2)
```

Comment:
an alternative to passing several arguments to `print` is string formatting via f-strings: https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python

for example:
```python
print(f"\nbase = {base}, height = {height}")
```